tests/config.js

```diff
@@ -0,0 +1,28 @@
+const dotenv = require( 'dotenv' );
+
+function config() {
+	dotenv.config();
+
+	verifyEnvVariables( [ 'AUTH_KEY', 'OWNER', 'REPO' ] );
+}
+
+function verifyEnvVariables( requiredVariables ) {
+	let anyMissingVariable =  false;
+
+	requiredVariables.forEach( variable => {
+		if ( !process.env[ variable ] ) {
+
+			console.log( chalk.red( `Missing ${ variable } env variable!` ) );
+
+			if ( !anyMissingVariable ) {
+				anyMissingVariable = true;
+			}
```

Comment:
```suggestion
			anyMissingVariable = true;
```

:point\_up: No need for `if` here I guess. Even if it's already `true` it will be just set once again. Alternatively you can go with:

```javascript
let allVariablesSet = true;
...
allVariablesSet = allVariablesSet && !process.env[ variable ];
```

but in this case it seems not necessary.